docs/development_guide.rst

```diff
@@ -2,7 +2,7 @@
 Development Guide
 =================
 
-This document gives an overview of the code in &&boolean.py&&, explaining the
+This document gives an overview of the code in `boolean.py`, explaining the
```

Comment:
Currently, it is just rendered verbatim as `&&boolean.py&&` so I changed it to render as a piece of code. See here up top: https://booleanpy.readthedocs.io/en/latest/development_guide.html